### ISSUE TITLE
12.4 R/ESS by Example(draft)

### ISSUE THREAD

**Issue by davidjayjackson** [emacs-ess/ESS#932]
Here is a draft of the **12.4 Example ESS**  page cleaned up and reformatted in M$ Word and Open Doc Format *.odf. 

Github repo:
https://github.com/davidjayjackson/EmacsRDocs

**Comment by vspinu** [comment]
I am not sure how useful the docx is. HTML and pdf are the preferred formats. The online section is [here](https://ess.r-project.org/Manual/ess.html#Example-ESS-usage).

I see now that there are a number of inaccurate and typose in those examples. Nor the listed shortcuts are any longer the recommended way to evaluate code these days. The most useful  `C-c C-c`, `C-M-x` and `C-RET` are not even listed there. 

Leaving this open as a doc  issue.

**Comment by davidjayjackson** [comment]
I uploaded a quick HTML doc( LibreOffice). I used the "online section" listed above as my source document.

**Comment by farcego** [comment]
I hope it may suit here as it is referenced for doc issues:
From the link  to the [online manual](https://ess.r-project.org/Manual/ess.html#Example-ESS-usage), I can read:
```
    ## Save the revised myfile.R with C-x C-s.
```
And a few lines below i read
```
    ## save the revised myfile.R when you have finished
    C-c C-s
```
Besides, I also read
```
 ## Save the entire *R* interaction buffer with C-c C-s
```
I reckon the middle one is a typo.
I am  running latest version (ess-version: 18.10.2 [Released git: d4cd65da6dbf]), and `C-c C-s` still have the behaviour of letting me choose which R session I want to switch to interact with (either from the active ones or by starting a new one).  Also, from `*R*` buffer,`C-c C-s` opens a new buffer named `*S search list*` with info about my session (file content pasted):
```
 [1] ".GlobalEnv"        "ESSR"              "package:stats"    
 [4] "package:graphics"  "package:grDevices" "package:utils"    
 [7] "package:datasets"  "package:methods"   "Autoloads"        
[10] "package:base"     
```
while C-x C-s would still save the R buffer into a file. So that could be a typo too?

Or maybe there has been ess changes after I updated it?

**Comment by vspinu** [comment]
Yes, those are the typos I was referring too.
